MiniApp/Classes/core/Models/MiniAppManifest.swift

```diff
@@ -54,6 +54,12 @@ public struct MiniAppManifest: Codable {
         self.optionalPermissions = optionalPermissions
         self.customMetaData = customMetaData
     }
+
+    public static func == (lhs: MiniAppManifest, rhs: MiniAppManifest) -> Bool {
+        lhs.requiredPermissions == rhs.requiredPermissions &&
+            lhs.optionalPermissions == rhs.optionalPermissions &&
+            lhs.customMetaData == rhs.customMetaData
```

Comment:
you should compare `Set`s (`Set(lhs.requiredPermissions) == Set(rhs.requiredPermissions)`) or `.sorted()` your arrays to be sure 2 manifests with permissions ordered differently are equal

Reply:
Done